pkg/storer/reserve.go

```diff
@@ -403,6 +403,9 @@ func (db *DB) unreserve(ctx context.Context) (err error) {
 
 // ReserveLastBinIDs returns all of the highest binIDs from all the bins in the reserve.
 func (db *DB) ReserveLastBinIDs() ([]uint64, error) {
+	if db.reserve == nil {
+		return make([]uint64, 0), nil
```

Comment:
Instead of allocating a new slice a `nil` should be returned.

Reply:
If I return nil, how will the following line handle that and will the /debugstore response still be correctly formed?   I know the nice empty slice works fine.

https://github.com/ethersphere/bee/blob/0ef39cd0857722e2eb55e9ddaf5a6dee3107d3c0/pkg/storer/debug.go#L150

Reply:
Not to mention this more-difficult-to-test use of this routine in case some peer requests cursors from an UltraLight node?   I know that's not SUPPOSED to happen, but still a node shouldn't panic if it does occur.

https://github.com/ethersphere/bee/blob/0ef39cd0857722e2eb55e9ddaf5a6dee3107d3c0/pkg/pullsync/pullsync.go#L549

Reply:
Changed to nil and verified that at least the /debugstore still works.
```
  "Reserve": {
    "Size": 0,
    "Capacity": 0,
    "LastBinIDs": null
  },
```